prism/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 The Rode Authors
```

Comment:
I'm not sure if I should put `The Rode Authors` here or if this should be a reference to the original creators of Prism?